plugins/catalog-backend/migrations/20201005122705_add_entity_full_name.js

```diff
@@ -21,19 +21,20 @@
  */
 exports.up = async function up(knex) {
   await knex.schema.alterTable('entities', table => {
-    table.text('full_name').nullable();
+    table.string('full_name').nullable();
   });
 
   await knex('entities').update({
     full_name: knex.raw(
-      "LOWER(kind) || ':' || LOWER(COALESCE(namespace, 'default')) || '/' || LOWER(name)",
+      "LOWER(??) || ':' || LOWER(COALESCE(??, 'default')) || '/' || LOWER(??)",
+      ['kind', 'namespace', 'name'],
     ),
   });
 
   // SQLite does not support alter column
   if (!knex.client.config.client.includes('sqlite3')) {
     await knex.schema.alterTable('entities', table => {
-      table.text('full_name').notNullable().alter({ alterNullable: true });
+      table.string('full_name').notNullable().alter();
```

Comment:
the alterNullable changes seem to be unnecessary - I tried running the tests with `{ alterNullable: true }` in place and they seemed to run fine. Could you try to see if you can revert that change throughout the PR?

Reply:
Yep! Changed. I think this is the only instance.